src/components/action-buttons.vue

```diff
@@ -61,12 +61,11 @@ const localOnSubmit = (event) => {
 }
 
 const goBack = () => {
-  window._paq?.push([
-    "trackEvent",
+  window.Piwik.getTracker().trackEvent(
```

Comment:
Pour les navigateurs récalcitrants ?
```suggestion
  window.Piwik?.getTracker().trackEvent(
```